### PR TITLE
Change the Compass runtime label

### PR DESCRIPTION
**Pull request by mvshao**
**Description**

Changes proposed in this pull request:

- change the invalid Compass label from `global_account_id` to `global_subaccount_id`

### CODE REVIEW

File: controllers/compassmanager_controller.go [hunk 1]
```diff
@@ -413,7 +413,7 @@ func createCompassRuntimeLabels(kymaLabels map[string]string) map[string]interfa
 	runtimeLabels["broker_instance_id"] = kymaLabels[LabelBrokerInstanceID]
 	runtimeLabels["gardenerClusterName"] = kymaLabels[LabelShootName]
 	runtimeLabels["subaccount_id"] = kymaLabels[LabelSubaccountID]
-	runtimeLabels["global_account_id"] = kymaLabels[LabelGlobalAccountID]
+	runtimeLabels["global_subaccount_id"] = kymaLabels[LabelGlobalAccountID]
```
Comment (Disper) [inline controllers/compassmanager_controller.go:416]: You're still taking the `LabelGlobalAccountID`, should it be a `LabelSubaccountID` instead?

**Review by Disper — CHANGES_REQUESTED** [review]

File: controllers/compassmanager_controller.go (hunk 1 above)
Comment (Disper) [reply controllers/compassmanager_controller.go:416]: don't know the logic in details and the variable naming sounds at least suspicious to me

Comment (mvshao) [reply controllers/compassmanager_controller.go]: I applied the changes

**Comment by Disper** [comment]
As this new label seems to be part of the contract, would it make sense to update `compassmanager_controller_test.go` so it will cover a scenario if this label is missing and fail in such case?

**Review by Disper — APPROVED** [review]
LGTM. I've also added `do-not-merge` as @koala7659 wanted to review it as well.

**Review by koala7659 — APPROVED** [review]
LGTM

**Pull request merged**